.env.example

```diff
@@ -74,6 +74,8 @@ CLOUDFRONT_DISTRIBUTION=
 COULDFRONT_SDK_VERSION=
 COULDFRONT_REGION=
 
+API_BASE_URL=https://api-dev.artic.edu
+API_PUBLIC_URL=https://api-dev.artic.edu
```

Comment:
If you don't set these in your local `.env`, then the app will [use the production api url](https://github.com/art-institute-of-chicago/artic.edu/blob/develop/config/api.php#L15-L26)!

Reply:
👍🏾